sdcm/nemesis.py

```diff
@@ -703,6 +703,18 @@ def _kill_scylla_daemon(self):
             self.target_node.wait_jmx_up()
         self.cluster.wait_for_schema_agreement()
 
+    def _sigquit_scylla_daemon(self):
```

Comment:
btw. we have already `generate_coredump_file` method in `BaseNode` class - this could reuse your new method and print diagnosis dump. 
Also add node log message with `DbNodeLogger` about this signal

Reply:
Updated this and other comments in new PR https://github.com/scylladb/scylla-cluster-tests/pull/10536